packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/EmptyState.tsx

```diff
@@ -0,0 +1,177 @@
+        maxWidth: GRID_MAX_WIDTH - 2 * GUTTER,
+        margin: '24px auto 0',
+      }}
+      direction="vertical"
+      gap={10}
+    >
+      {isPersonalSpace ? (
+        <StyledEmptyDescription as="p">
+          Sandboxes are a great way to prototype your ideas with zero startup
+          costs and with everything you need: a code editor, previews, dev
+          servers, unit tests, Storybook and many other devtools.
+        </StyledEmptyDescription>
+      ) : (
+        <Stack
+          css={{
+            width: '100%',
+            '@media screen and (min-width: 768px)': {
+              width: '560px',
+            },
+          }}
+          direction="vertical"
+          gap={6}
+        >
+          <StyledEmptyDescription as="p">
+            Sandboxes are a great way to prototype your ideas with zero startup
+            costs and with everything you need: a code editor, previews, dev
+            servers, unit tests, Storybook and many other devtools.
+          </StyledEmptyDescription>
+          <StyledEmptyDescription as="p">
+            Sandboxes in this section are visible to all team members.
+          </StyledEmptyDescription>
+        </Stack>
+      )}
+      <Stack direction="vertical" gap={6}>
+        <Text as="h2" margin={0} lineHeight="25px" size={16} weight="normal">
+          Start from a template
+        </Text>
+        <Element
+          as="ul"
+          css={{
+            margin: 0,
+            padding: 0,
+            position: 'relative',
+            overflow: 'hidden',
+            display: 'grid',
+            listStyle: 'none',
+            gap: '16px',
+            gridTemplateColumns: 'repeat(auto-fill, minmax(268px, 1fr))',
+            gridAutoRows: 'minmax(156px, 1fr)',
+          }}
+        >
+          {officialTemplates.state === 'loading' &&
+            new Array(4).fill(undefined).map((_, i) => (
+              <SkeletonText
+                // eslint-disable-next-line react/no-array-index-key
+                key={`templates-skeleton-${i}`}
+                css={{
+                  width: '100%',
+                  height: '100%',
+                }}
+              />
+            ))}
+          {officialTemplates.state === 'ready' &&
+            officialTemplates.templates.length > 0 &&
+            officialTemplates.templates
+              .filter(t => TEMPLATE_IDS.includes(t.sandbox.id))
+              .map(template => (
+                <Stack as="li" key={template.id}>
+                  <TemplateCard
+                    key={template.id}
+                    disabled={hasMaxPublicSandboxes}
+                    template={template}
+                    onOpenTemplate={handleOpenTemplate}
+                    onSelectTemplate={handleSelectTemplate}
+                  />
+                </Stack>
+              ))}
+
+          {((officialTemplates.state === 'ready' &&
+            officialTemplates.templates.length === 0) ||
```

Comment:
shouldn't we show the option to create from template (which opens the create modal) for all scenarios? as the last card here.

Reply:
@alexnm that's not in the designs but I can ask Lena.

Reply:
Yes, just a thought, you have the option to create a sandbox always from the context menu